cmake/vsconfig.cmake

```diff
@@ -0,0 +1,23 @@
+#
+# Extra configuration for Visual Studio Generator
+#
+
+# detect if we're setting up a visual studio solution
+if(CMAKE_GENERATOR MATCHES "^Visual Studio.*$")
+    set(VCPROJ ON)
+endif()
+
+# add user props to build directory to add to project settings
+# for running targets in the IDE
+if(VCPROJ)
+    file(GENERATE
+        OUTPUT "${CMAKE_BINARY_DIR}/user.props"
+        CONTENT "<?xml version='1.0' encoding='utf-8'?>
+<Project ToolsVersion='15.0' xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
```

Comment:
This looks fair specific to a single version of MSVC. Is that not the case?

Reply:
Right.  And what I don't get is why we'd be using PDAL_PLUGIN_INSTALL_PATH.  The issue was finding DLLs from the debugger env.  There was talk of using the conda PATH.  This makes sense, but I'm not sure what the PDAL_PLUGIN_INSTALL_PATH has to do with a PATH you would use to find libgdal.dll.  The assumption here is that they're the same, I think.  That hasn't been the case for me when I've used VS.  I haven't even set the install path.  I looked for some ENV VAR that conda sets as its part of the PATH env, but didn't find one.  And even in that case, it's env. specific.  You might not be using conda.  I guess this just doesn't seem like a PDAL build issue.  It seems like a general environment issue.  Maybe someone should make a program that creates this `user.props` given a set of paths that you run for your environment.

All that said, I could be misunderstanding.